### PR TITLE
FRC branding

### LINKED ISSUE #250: Correct FRC acronym branding
The registered trademark symbol should be added to the first use of the FRC acronym in body text on any page where it appears.

This can be accomplished by using

`..include:: <isonum.txt>` at the top of the document and the using `FRC\ |reg|` at the first usage of the acronym

### PR DESCRIPTION
**Pull request by jasondaming**
Closes #250 

That was a lot of corrections!

### CODE REVIEW

File: source/docs/contributing/frc-docs/build-instructions.rst [hunk 1]
```diff
@@ -1,7 +1,9 @@
+.. include:: <isonum.txt>
+
 Build Instructions
 ==================
 
-This document contains information on how to build the HTML, PDF, and EPUB versions of the frc-docs site. frc-docs uses Sphinx as the documentation generator. This document also assumes you have basic knowledge of `Git <https://git-scm.com/>`__ and console commands.
+This document contains information on how to build the HTML, PDF, and EPUB versions of the frc\ |reg|-docs site. frc-docs uses Sphinx as the documentation generator. This document also assumes you have basic knowledge of `Git <https://git-scm.com/>`__ and console commands.
```
Comment (sciencewhiz) [inline source/docs/contributing/frc-docs/build-instructions.rst:6]: frc-docs is a name. It shouldn't have the registered trademark symbol inserted

File: source/docs/software/vscode-overview/wpilib-commands-vscode.rst [hunk 2]
```diff
@@ -33,7 +35,7 @@ To access these commands, press Ctrl+Shift+P to open the Command Palette, then b
 - **WPILib: Reset Ask for WPILib Updates Flag** - This will clear the flag on the current project, allowing you to re-prompt to update a project to the latest WPILib version if you previously chose to not update.
 - **WPILib: Run a command in Gradle** - This lets you run an arbitrary command in the GradleRIO command environment
 - **WPILib: Set Team Number** - Used to modify the team number associated with a project. This is only needed if you need to change the team number from the one initially specified when creating the project.
-- **WPILib: Set VS Code Java Home to FRC Home** - Set the VS Code Java Home variable to point to the Java Home discovered by the FRC extension. This is needed if not using the offline installer to make sure the intellisense settings are in sync with the WPILib build settings.
+- **WPILib: Set VS Code Java Home to FRC\ |reg| Home** - Set the VS Code Java Home variable to point to the Java Home discovered by the FRC extension. This is needed if not using the offline installer to make sure the intellisense settings are in sync with the WPILib build settings.
```
Comment (sciencewhiz) [inline source/docs/software/vscode-overview/wpilib-commands-vscode.rst:38]: This is a title and shouldnt have the registered trademark symbol

**Review by sciencewhiz — CHANGES_REQUESTED** [review]

**Review by sciencewhiz — DISMISSED** [review]

**Comment by Daltz333** [comment]
Titles should not have the registered trademark. This breaks translation.

**Comment by jasondaming** [comment]
branding from titles is removed

**Review by Daltz333 — APPROVED** [review]

**Pull request merged**